### ISSUE TITLE
Would it be possible to use docker ports instead...

### ISSUE THREAD

**Issue by dreams-and-thoughts** [mporium/GlueZeppelin#2]
I like your solution, but I was wondering if it would be possible to use docker ports instead of relying on your dockerfile, and staying up to data...

eg.

`docker run -p 8080:8080 -p 9007:9007 --rm -v $PWD/logs:/logs -v $PWD/notebook:/notebook -e ZEPPELIN_LOG_DIR='/logs' -e ZEPPELIN_NOTEBOOK_DIR='/notebook' --name zeppelin apache/zeppelin:0.8.1`

And then create the SSH tunnel on the host machine, instead of in the container:

`ssh -i private-key.pem -vnNT -L localhost:9007:169.254.76.1:9007 glue@ec2-13-211-47-17.ap-southeast-2.compute.amazonaws.com`

What do you think? My knowledge of docker and zeppelin is limited, and I haven't been able to get it working... Zeppelin says Connection Refused.

**Comment by amilbourne** [comment]
I don't see why your plan shouldn't work.  I may have done that myself at one point but I can't remember now.

I mostly used this page to figure out how to get this working:
https://docs.aws.amazon.com/glue/latest/dg/dev-endpoint-tutorial-local-notebook.html

In there is says that Zeppelin 0.8.x is not supported and 0.7.3 should be used.  I notice that you reference 0.8 in the example above.  Have you tried 0.7.3?

**Comment by dreams-and-thoughts** [comment]
Hey, thanks for responding. I eventually did get this working using Docker ports, and I _think_ the issue was the Zeppelin version, as you guessed. 

In case anyone else stumbles on this, after getting Zeppelin running, I had to manually edit the ZEPPELING_HOME/conf/interpreter.json file:

```
"master": "yarn-client",

"option": {
        "remote": true,
        "port": 9007,
        "host": "localhost",
```

Zeppelin 0.7.3 interpreter settings page seemed to be broken, and wouldn't allow editing of the variables... for me at least.
